Modules/Loadable/Transforms/MRMLDM/vtkMRMLLinearTransformsDisplayableManager2D.cxx

```diff
@@ -233,7 +233,7 @@ void vtkMRMLLinearTransformsDisplayableManager2D::vtkInternal::RemoveTransformNo
 }
 
 //---------------------------------------------------------------------------
-void vtkMRMLLinearTransformsDisplayableManager2D::vtkInternal::UpdateDisplayableTransforms(vtkMRMLTransformNode* mNode)
+void vtkMRMLLinearTransformsDisplayableManager2D::vtkInternal::UpdateDisplayableTransforms(vtkMRMLTransformNode* vtkNotUsed(mNode))
```

Comment:
@Sunderlandkyl Should the implementation of this be similar to the one of `vtkMRMLLinearTransformsDisplayableManager3D` ?

See https://github.com/Slicer/Slicer/blob/main/Modules/Loadable/Transforms/MRMLDM/vtkMRMLLinearTransformsDisplayableManager3D.cxx#L260-L275